client/ayon_fusion/plugins/create/create_saver.py

```diff
@@ -56,7 +56,8 @@ def get_pre_create_attr_defs(self):
 
     def _get_frame_range_enum(self):
         frame_range_options = {
-            "current_folder": "Current Folder context",
+            # The `current_folder` value is kept for backwards compatibility
```

Comment:
And can we change it? Like, this is uber confusing.

Reply:
We could - but it'd also mean updating in `settings` as value (which I suppose is doable?) because settings also use it to define a default value.

However, it looks like [that default value may actually be broken because it seems it was not refactored along.](https://github.com/ynput/ayon-fusion/blob/94eac3e658b3e4ac67aa1e0fa07d6e7b93c25b81/server/settings.py#L40) so with that in mind it'd be broken anyway?

Reply:
Oh, crap... Worse than I expected. In that case, can you change it.

Reply:
Done with https://github.com/ynput/ayon-fusion/pull/9/commits/18f3b95fdcd292b91af9b47a45e00f8e2c531308